opus/application/static_media/js/cart.js

```diff
@@ -539,6 +539,9 @@ var o_cart = {
                     }
                 }
             });
+        } else {
+            // Make sure "Add all results to cart" is hidden in cart tab.
+            $("#op-obs-menu .dropdown-item[data-action='addall']").addClass("op-hide-element");
         }
```

Comment:
Yes, but because every time we go to browse tab (call activateBrowseTab, and loadData is called as well in the function), the icon will be shown. And if we switch back to cart tab (from browse tab) again without reloading obs in cart tab, the icon will not be hidden (loadData is not called in activateCartTab). Therefore, we hide the icon here for the case that obs in cart tab is not reloaded.